packages/advanced-logic/src/advanced-logic.ts

```diff
@@ -124,6 +130,11 @@ export default class AdvancedLogic implements AdvancedLogicTypes.IAdvancedLogic
           `extension with id: ${id} not found for network: ${requestState.currency.network}`,
         );
       }
+
+      if (id === ExtensionTypes.ID.PAYMENT_NETWORK_ANY_TO_NATIVE_TOKEN) {
```

Comment:
I think this can be slightly refactored into a more elegant way (check the block before)